PWGHF/D2H/DataModel/ReducedDataModel.h

```diff
@@ -370,6 +370,35 @@ DECLARE_SOA_TABLE(HfCandBpConfigs, "AOD", "HFCANDBPCONFIG", //! Table with confi
                   hf_cand_bplus_config::MySelectionFlagD0,
                   hf_cand_bplus_config::MySelectionFlagD0bar,
                   hf_cand_bplus_config::MyInvMassWindowD0Pi);
+
+// Charm resonances analysis
+namespace hf_reso_cand_reduced
+{
+DECLARE_SOA_COLUMN(InvMass, invMass, float); //! Invariant mass of V0/D candidate in GeV/c2
+DECLARE_SOA_COLUMN(Px, px, float);           //! Momentum of V0/D candidate in GeV/c
+DECLARE_SOA_COLUMN(Py, py, float);
+DECLARE_SOA_COLUMN(Pz, pz, float);
+} // namespace hf_reso_cand_reduced
+
+DECLARE_SOA_TABLE(HfRedVzeros, "AOD", "HFREDVZERO", //! Table with V0 candidate information for resonances reduced workflow
+                  o2::soa::Index<>,
+                  hf_track_index_reduced::Prong0Id, hf_track_index_reduced::Prong1Id,
+                  hf_track_index_reduced::HfRedCollisionId,
+                  hf_cand::XSecondaryVertex, hf_cand::YSecondaryVertex, hf_cand::ZSecondaryVertex,
+                  hf_reso_cand_reduced::InvMass,
```

Comment:
Depending on the cuts applied, a V0 can be compatible with more than one hypothesis (K0S, Lambda, AntiLambda). Even if currently you implemented only one channel, I would suggest to add the three columns here, as well as a bitmap (that can be `uint8_t`) that has the information of the mass hypothesis (K0S, Lambda, or AntiLambda), so that it will be easier to add the others later on.

Reply:
Ok!